docker/bin/zookeeperStart.sh

```diff
@@ -88,17 +88,23 @@ if [[ "$WRITE_CONFIGURATION" == true ]]; then
     echo $MYID > $MYID_FILE
     echo "server.${MYID}=${ZKCONFIG}" > $DYNCONFIG
   else
+    set -ex
+    ZKURL=$(zkConnectionString)
+    CONFIG=`java -Dlog4j.configuration=file:"$LOG4J_CONF" -jar /root/zu.jar get-all $ZKURL`
     echo Writing configuration gleaned from zookeeper ensemble
     echo "$CONFIG" | grep -v "^version="> $DYNCONFIG
```

Comment:
this is already done in REGISTER step, see https://github.com/pravega/zookeeper-operator/pull/129